### PR TITLE
Doc update: list FDE prereq

### PR DESCRIPTION
**Pull request by sabaini**

### CODE REVIEW

File: docs/explanation/fde-osd.rst [hunk 1]
```diff
@@ -26,6 +26,16 @@ Limitations
 **Warning:** It is important to note that MicroCeph FDE *only* encompasses OSDs. Other data, such as state information for monitors, logs, configuration etc., will *not* be encrypted by this mechanism.
 
 
+Prerequisites
```
Comment (UtkarshBhatthere) [inline docs/explanation/fde-osd.rst:29]: Minor Nit: I think the pre-requisites section should be moved up maybe just after the overview.

**Review by UtkarshBhatthere — CHANGES_REQUESTED** [review]
Just a minor nit with placement of the section. Unrelated to this particular change.... I feel we need to do some diataxis segregation for FDE doc but tbh I have no idea how that would look like.

File: docs/explanation/fde-osd.rst (hunk 1 above)
Comment (sabaini) [reply docs/explanation/fde-osd.rst:23]: Ack, reasonable

**Comment by sabaini** [comment]
Not wrong wrt to the Diataxis split. I'll add that to the todo list

**Comment by UtkarshBhatthere** [comment]
Ship it! 🛳️

**Pull request merged**